modules/announcements.lua

```diff
@@ -108,7 +81,7 @@ function Announcements:UNIT_AURA(event, unit)
 	if not self:IsHandledUnit(unit) or not self.db[unit].drinks then return end
 
 	for i = 1, 40 do
-		local name, _, _, _, _, _, _, _, _, spellID = UnpackAuraData2(UnitBuff(unit, i, "HELPFUL"))
+		local name, _, _, _, _, _, _, _, _, spellID = GladiusEx:UnitBuff(unit, i, "HELPFUL")

```

Comment:
This should be `GladiusEx.UnitBuff` (`a:b(c)` is just a shorthand for `a.b(a, c)`)

Reply:
thanks, fixing now